pom.xml

```diff
@@ -1290,13 +1290,13 @@
     <rancidApiVersion>1.0.4</rancidApiVersion>
     <rateLimitedLoggerVersion>1.1.0</rateLimitedLoggerVersion>
     <resilience4jVersion>0.17.0</resilience4jVersion>
-    <rocksdbjniVersion>5.15.10</rocksdbjniVersion>
+    <rocksdbjniVersion>5.18.3</rocksdbjniVersion>
     <scalaLibraryVersion>2.12.8</scalaLibraryVersion>
     <scalaLoggingVersion>3.9.0</scalaLoggingVersion>
     <scalaVersion>2.12</scalaVersion>
     <servicemixSpecsVersion>2.9.0</servicemixSpecsVersion>
     <servletApiVersion>3.1.0</servletApiVersion>
-    <slf4jVersion>1.7.21</slf4jVersion>
+    <slf4jVersion>1.7.26</slf4jVersion>
```

Comment:
Don't see this is getting used in kafka related features. Any other reason this is upgraded ?

Reply:
> Smoke tests seems to be failing. Can you take a look once ?

Smoke failed in CircleCI due to an issue with CircleCI, and/or a known issue with Karaf loading dependencies. The smoke passed on Bamboo, didn't want to waste the minutes triggering it again on the off chance it fails due to the CircleCI/Karaf known issues.

Reply:
> Don't see this is getting used in kafka related features. Any other reason this is upgraded ?

See: https://search.maven.org/artifact/org.apache.servicemix.bundles/org.apache.servicemix.bundles.kafka_2.12/2.3.0_1/bundle

Pom lists that version in the bundle, this was my reason for upgrading that particular version.

Reply:
Since all smoke tests are passing, I would think it is safe.